zoom_monitor.py

```diff
@@ -148,20 +148,10 @@ def _open_participants_list(self):
         except NoSuchElementException:
             pass  # We need to open it.
 
-        # Sometimes, the controls to open the participants list get hidden
-        # (for example, when someone stops sharing their screen, maybe?). In
-        # order to make the controls visible again, move the mouse. We move
-        # the mouse by finding a DOM component that should always be on the
-        # screen, and then moving the mouse to two locations near it.
-        main_element_id = "wc-content"
-        self._wait_for_element(By.ID, main_element_id)
-        main_element = self._driver.find_element(By.ID, main_element_id)
-        ActionChains(self._driver)\
-            .move_to_element_with_offset(main_element, 0, 0)\
-            .move_to_element_with_offset(main_element, 1, 1)\
-            .perform()
-
-        self._wait_for_element(By.CLASS_NAME, "SvgParticipantsDefault")
+        element = self._wait_for_element(
+            By.XPATH, "//*[contains(@class, 'SvgParticipants')]")
```

Comment:
We need to put in a comment explaining why this is here. Something like:
> When starting up, we're looking for a button in the class `SvgParticipantsDefault`. However, if we need to double-click on something (see below), the class of the button will change to `SvgParticipantsHovered`. Since we're looking for either one, we wait for any element whose class starts with `SvgParticipants`. 

The part about "see below" needs some rephrasing, but it'll be a comment kinda-sorta like that.

Reply:
Over documenting this might be a problem too, as we'd be explaining the same thing four times. I'd prefer to keep it to one clear statement.